source/documentation/how-to-release.html.md

```diff
@@ -90,24 +91,23 @@ Hi everyone,
 I propose the following RC to be released as official Apache Parquet <VERSION> release.
 
 The commit id is <SHA1>
-* This corresponds to the tag: apache-parquet-<VERSION>
-* https://github.com/apache/parquet-format/tree/<SHA1>
-* https://git-wip-us.apache.org/repos/asf/projects/repo?p=parquet-mr.git&a=commit&h=<SHA1>
+* This corresponds to the tag: apache-parquet-<VERSION>-rc<NUM>
+* https://github.com/apache/parquet-mr/tree/<SHA1>
 
 The release tarball, signature, and checksums are here:
 * https://dist.apache.org/repos/dist/dev/parquet/<PATH>
 
 You can find the KEYS file here:
-* https://dist.apache.org/repos/dist/dev/parquet/KEYS
+* https://apache.org/dist/parquet/KEYS
 
 Binary artifacts are staged in Nexus here:
-* https://repository.apache.org/content/groups/staging/org/apache/parquet/parquet/
+* https://repository.apache.org/content/groups/staging/org/apache/parquet/
 
 This release includes important changes that I should have summarized here, but I'm lazy.
 
 Please download, verify, and test.
 
-Please vote by <72 HOUR FROM NOW>
+Please vote in the next 72 hours.
```

Comment:
I think we usually phrase it as "The vote will be open for at least 72 hours." so that we don't have to fail the release if people are slow to vote.

Reply:
I think that "Please vote in the next 72 hours" does not mean the vote will be closed after 72 hours. It only requests members to vote in 72 hours so the vote can be closed "soon".
I'm afraid the meaning of the suggested one might be "I don't have to hurry to vote because it won't be close even after 72 hours."